pkg/util/configdiag.go

```diff
@@ -0,0 +1,15 @@
+package util
```

Comment:
I had to make this bit public because we use it inside the public sendconfig package (specifically `pkg/sendconfig/common_workflows.go`). I think it's only public now because it's used in both 1.x and 2.x, and refactoring any of the shared stuff is a minefield for the time being.

Reply:
Ok, I created a follow up item to review and privatize the remaining bits before 2.0:

https://github.com/Kong/kubernetes-ingress-controller/issues/1591

We can follow up there with some cleanup.